src-docs/src/views/theme/other/other.tsx

```diff
@@ -168,25 +168,6 @@ export default () => {
 
       <EuiSpacer size="l" />
 
-      <EuiText grow={false}>
-        <p>
```

Comment:
We should actually keep this text in this file because it will show for both JS and Sass languages.